src/coreComponents/linearAlgebra/interfaces/hypre/mgrStrategies/CompositionalMultiphaseHybridFVM.hpp

```diff
@@ -70,9 +70,9 @@ class CompositionalMultiphaseHybridFVM : public MGRStrategyBase< 3 >
 
     setupLabels();
 
-    m_levelFRelaxMethod[0] = 0; // Jacobi
-    m_levelFRelaxMethod[1] = 0; // Jacobi
-    m_levelFRelaxMethod[2] = 0; // Jacobi
+    m_levelFRelaxMethod[0] = hypre::getAMGRelaxationType( LinearSolverParameters::AMG::SmootherType::jacobi );
+    m_levelFRelaxMethod[1] = hypre::getAMGRelaxationType( LinearSolverParameters::AMG::SmootherType::jacobi );
+    m_levelFRelaxMethod[2] = hypre::getAMGRelaxationType( LinearSolverParameters::AMG::SmootherType::jacobi );
```

Comment:
I think you're assigning the wrong thing here, relaxation type (0..18) to relaxation method ({0,1,2}). The old comment `// Jacobi` was probably also incorrect?

Reply:
You're correct. Here we need to specify the F-Relaxation method, i.e. `singleLevel`, `multilevel`, or `amgVCycle`. We would need to specify the F-Relaxation type -- in this case `jacobi`) -- introducing a new member array `m_levelFRelaxType` in `MGRStrategyBase`. However, we realized that such capability is currently missing in MGR. @oseikuffuor1 is working on it.